src/common/text.c

```diff
@@ -359,7 +359,7 @@ const RegionName kRegionNamesOot[] = {
     { "in",         "the " TEXT_COLOR_YELLOW "Sacred Realm" },
     { "inside",     "the " TEXT_COLOR_GREEN "Deku Tree" },
     { "inside",     TEXT_COLOR_RED "Dodongo's Cavern" },
-    { "inside",     TEXT_COLOR_BLUE "Jabu-Jabu" },
+    { "inside",     TEXT_COLOR_BLUE "Jabu Jabu's Belly" },
```

Comment:
The game has the hyphen there

Reply:
Wouldn't be on par with OOTR hints, but I suppose - [main OOTR hints code](https://github.com/TestRunnerSRL/OoT-Randomizer/blob/Dev/Hints.py#L330)